frontend/src/utils/api/Project.ts

```diff
@@ -0,0 +1,32 @@
+import apiFetch from "../ApiFetch.ts";
+import {ProjectInput} from "../InputInterfaces.ts";
+import {Backend_group, Backend_Project} from "../BackendInterfaces.ts";
+import {mapGroup} from "../ApiTypesMapper.ts";
+import {Group} from "../ApiInterfaces.ts";
+
+export async function project_create_group(project_id: number): Promise<Group> {
+    const groupData = await apiFetch<Backend_group>(`/projects/${project_id}/groups`, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    });
+    if (groupData.ok){
+        return mapGroup(groupData.content);
+    }else{
```

Comment:
Je returned al in de if, dus de else is niet nodig.
throw gewoon altijd na de if.